redfish-core/lib/log_services.hpp

```diff
@@ -2836,10 +2852,17 @@ inline void requestRoutesDBusCELogEntry(App& app)
                 return;
             }
 
+            if (hidden == nullptr)
+            {
+                // Skip log entry if Hidden dbus property is missing
+                messages::resourceNotFound(asyncResp->res, "LogEntry",
```

Comment:
Again.. Not sure resourceNotFound is correct here.. InteneralError ? Oh well.. 
Also we could trace it..